app/models/checklist_template.rb

```diff
@@ -0,0 +1,2 @@
+class ChecklistTemplate < ApplicationRecord
+end
```

Comment:
I *think* if we add a table to the db for checklist templates (which we should), then our `annotate` gem will automatically populate the top of this file with some useful db field info.